fv3core/stencils/fvtp2d.py

```diff
@@ -228,25 +241,17 @@ def __call__(
             self._tmp_q_j,
         )
         self.y_piecewise_parabolic_outer(self._tmp_q_j, cry, fy)
-        if mfx is not None and mfy is not None:
-            self.stencil_transport_flux(
-                fx,
-                self._tmp_fx2,
-                fy,
-                self._tmp_fy2,
-                mfx,
-                mfy,
-            )
-            if (mass is not None) and self.delnflux is not None:
-                self.delnflux(q, fx, fy, mass=mass)
```

Comment:
The part about passing in aliases for xfx and yfx, assigning mfx and mfy to help reduce code duplication makes sense. But this particular part of the removed logic is a little more easy to miss, where the mass needs to be present to call delnflux when mfx and mfy are mass fluxes. This does warrant some explanation I think to help a user out.